packages/code-editor/src/Panel/Panel.tsx

```diff
@@ -125,100 +129,120 @@ export function Panel({
     onDownloadClick?.();
   };
 
+  const handleViewShortcutsClick = () => {
+    setShortcutsModalOpen(true);
+    onViewShortcutsClick?.();
+  };
+
   return (
-    <div className={getPanelStyles(theme)}>
-      <div
-        className={getPanelTitleStyles(theme, baseFontSizeProp || baseFontSize)}
-      >
-        {title}
-      </div>
-      <div className={getPanelInnerContentStyles()}>{innerContent}</div>
-      <div className={getPanelButtonsStyles()}>
-        {showFormatButton && (
-          <Tooltip
-            align="top"
-            justify="middle"
-            trigger={
-              <IconButton onClick={handleFormatClick} aria-label="Format code">
-                <FormatIcon />
-              </IconButton>
-            }
-            triggerEvent="hover"
-            darkMode={darkMode}
-          >
-            Prettify code
-          </Tooltip>
-        )}
-        {showCopyButton && (
-          <CodeEditorCopyButton
-            variant={CopyButtonVariant.IconButton}
-            getContentsToCopy={getContents ?? (() => '')}
-            onCopy={onCopyClick}
-          />
-        )}
-        {showSecondaryMenuButton && (
-          <Menu
-            trigger={
-              <IconButton aria-label="Show more actions">
-                <EllipsisIcon />
-              </IconButton>
-            }
-            variant={MenuVariant.Compact}
-            darkMode={darkMode}
-            renderDarkMenu={false}
-          >
-            <MenuItem
-              glyph={<UndoIcon />}
-              onClick={handleUndoClick}
-              aria-label="Undo changes"
-            >
-              Undo
-            </MenuItem>
-            <MenuItem
-              glyph={<RedoIcon />}
-              onClick={handleRedoClick}
-              aria-label="Redo changes"
-            >
-              Redo
-            </MenuItem>
-            <MenuItem
-              glyph={<DownloadIcon />}
-              onClick={handleDownloadClick}
-              aria-label="Download code"
+    <>
+      <div className={getPanelStyles(theme)}>
+        <div
+          className={getPanelTitleStyles(
+            theme,
+            baseFontSizeProp || baseFontSize,
+          )}
+        >
+          {title}
+        </div>
+        <div className={getPanelInnerContentStyles()}>{innerContent}</div>
+        <div className={getPanelButtonsStyles()}>
+          {showFormatButton && (
+            <Tooltip
+              align="top"
+              justify="middle"
+              trigger={
+                <IconButton
+                  onClick={handleFormatClick}
+                  aria-label="Format code"
+                >
+                  <FormatIcon />
+                </IconButton>
+              }
+              triggerEvent="hover"
+              darkMode={darkMode}
```

Comment:
nit: I think the default for `Tooltip` is `hover`, so you don't need to specify it. Also, is `CodeEditor` wrapped in a `LeafyGreenProvider`? If so, you probably don't need to pass `darkMode` to `Tooltip`.